### DISCUSSION TITLE
Example issues

### DISCUSSION THREAD

**Discussion by elritz** [enisdenjo/graphql-sse#40] [Q&A]
**Screenshot**
Visualising is always helpful.
![Screenshot 2022-12-07 at 2 33 58 PM](https://user-images.githubusercontent.com/20426361/206278282-08683f52-faaa-4c53-9b50-5ba14a9d429e.png)
[Codegen image]
This is a codegen file which is connecting to the endpoint where the server with the sse handler is at on endpoint /graphql/stream.
 * An issue that im seeing here is that the client isnt able to connect for that route. 
 And there might not be a schema available!
Even routing to this page through the browser its telling me it's not what i think to expect here, I know it works because other routes operate fine: 
![Screenshot 2022-12-07 at 2 36 06 PM](https://user-images.githubusercontent.com/20426361/206278699-9bf91172-0058-4f2a-92e2-36829b064549.png)
[Chrome /graphql/stream image]

Here is the server running, it is connecting to "/" where i just test to see the h2  in my Network inspector. According to the documentation. 
![Screenshot 2022-12-07 at 2 40 40 PM](https://user-images.githubusercontent.com/20426361/206279625-4fa76f08-d8d3-4cb8-8d71-2c0bda432384.png)

Im also unsure about the situation on the client side. I've copied the code exactly and removed the headers part for my own purposes!  
![Screenshot 2022-12-07 at 2 44 08 PM](https://user-images.githubusercontent.com/20426361/206280266-16b69ffb-1c25-488a-a4c0-89961ccbf96e.png)
[Client Side SSELink]

![Screenshot 2022-12-07 at 2 44 49 PM](https://user-images.githubusercontent.com/20426361/206280401-8c3241b0-7567-4ac0-8022-e15672fced9b.png)
[Final Combination for Apollo provider]

https://www.apollographql.com/docs/react/api/link/introduction/#directional-composition
[Link to Apollo Documentation on linking]

From reading the Apollo link documentation, A directional integration is appropriate, 
![Screenshot 2022-12-07 at 3 06 47 PM](https://user-images.githubusercontent.com/20426361/206284567-6bc72a7e-71d9-494a-9619-409674e5273d.png)
[SSE integration using a Direction Link] (concept not working)

**Expected Behaviour**
I expected it to do _this_ -

**Actual Behaviour**
Im unsure what the server is suppose to return from the handler, but what i expect is probably a schema but currently just unable to connect from the client 

**Debug Information**
Help us debug the bug?

**Further Information**
This is a React Native project, i know mobile accepts the h2 unsure if its relevant to the sse package!
In the example with the Apollo Client I think it maybe incorrect for h2 because the link isnt using http2!
I understand im not asking a specific question but my real issue it i can't seem to make it work for me! 
If we can make it work i would be happy to provide that update.

**Comment by enisdenjo** [comment]
This feels like an integration issue, not specific to graphql-sse. Are you sure your codegen gateway supports graphql-sse? What errors do you get downstream?
